CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Version <dev> of the agent adds the ability to filter logs by level and expands
     * `transmit_subscription_confirmation.action_cable`
     * `transmit_subscription_rejection.action_cable`
 
+- **Bugfix: Report Code Level Metrics for Rails controller methods**
```

Comment:
```suggestion
- **Bugfix: Report Code-Level Metrics for Rails controller methods**
```

Reply:
@hannahramadan - Could you add this to the overview description at the top of the Changelog?